sdk/storage/azure-storage-blob/README.md

```diff
@@ -1,15 +1,217 @@
+# Azure Storage Blobs client library for Python
+Azure Blob storage is Microsoft's object storage solution for the cloud. Blob storage is optimized for storing massive amounts of unstructured data, such as text or binary data.
 
+Blob storage is ideal for:
+
+* Serving images or documents directly to a browser
+* Storing files for distributed access
+* Streaming video and audio
+* Storing data for backup and restore, disaster recovery, and archiving
+* Storing data for analysis by an on-premises or Azure-hosted service
+
+[Source code](TODO) | [Package (PyPi)](TODO) | [API reference documentation](https://docs.microsoft.com/en-us/rest/api/storageservices/blob-service-rest-api) | [Product documentation](https://docs.microsoft.com/en-us/azure/storage/) | [Samples](TODO)
 
-# Azure Storage Blobs client library for Python
 
 ## Getting started
 
+### Install the package
+Install the Azure Storage Blobs client library for Python with [pip](https://pypi.org/project/pip/):
+
+```bash
+pip install azure-storage-blobs
+```
+
+**Prerequisites**: You must have an [Azure subscription](https://azure.microsoft.com/en-us/free/), and a
+[Storage Account](https://docs.microsoft.com/en-us/azure/storage/common/storage-account-overview) to use this package.
+
+To create a Storage Account, you can use the [Azure Portal](https://docs.microsoft.com/en-us/azure/storage/common/storage-quickstart-create-account?tabs=azure-portal),
+[Azure PowerShell](https://docs.microsoft.com/en-us/azure/storage/common/storage-quickstart-create-account?tabs=azure-powershell) or [Azure CLI](https://docs.microsoft.com/en-us/azure/storage/common/storage-quickstart-create-account?tabs=azure-cli):
+
+```bash
+az storage account create -n MyStorageAccountName -g MyResourceGroupName
+```
+
+Requires Python 2.7, 3.5 or later to use this package.
+
+### Authenticate the client
+
+Interaction with Storage Blobs starts with an instance of the BlobServiceClient class. You need an existing storage account, its URL, and a credential to instantiate the client object.
+
+#### Get credentials
+
+To authenticate the client you have a few options:
+1. Use a SAS token string 
+2. Use an account shared access key
+3. Use a token credential from azure.identity
```

Comment:
We should make a note to upadte "azure.identity" to a link to that package once it's published on PyPI (hopefully monday)